controllers/route/route_controller.go

```diff
@@ -233,11 +233,9 @@ func (e *enqueueRequestForReferecingRoutes) matchSecret(c client.Client, secret
 	for _, route := range routeList.Items {
 		if secretName := route.GetAnnotations()[certAnnotation]; secretName == secret.Name && route.Spec.TLS != nil {
 			result = append(result, route)
-			break
```

Comment:
should this be a `continue`?, otherwise we risk returning a route twice.

Reply:
yes because they could be tagged with both annotations good catch, i'll make the change